src/test/javascript/spec/component/course/course-overview.component.spec.ts

```diff
@@ -235,6 +237,8 @@ describe('CourseOverviewComponent', () => {
                 findAllForDropdownSpy = jest
                     .spyOn(courseService, 'findAllForDropdown')
                     .mockReturnValue(of(new HttpResponse({ body: coursesDropdown, headers: new HttpHeaders() })));
+                printService = TestBed.inject(ThemeService);
+                (printService as any).isNavbarCollapsed$ = of(true);
```

Comment:
If this is not generally set to collapsed the test for the navbar should still work, we should only hide the navbar for the print test case

Reply:
removed 👍